sample.coffee

```diff
@@ -19,14 +19,17 @@ myData = FileCollection({
 
 if Meteor.isClient
 
+   # This assigns a file drop zone to the "file table"
+   # once DOM is ready so jQuery can see it
+   Template.collTest.onRendered ->
```

Comment:
I understand why putting this in `onRendered` is good, but shouldn't this still be inside the `Meteor.startup` below? Until that callback, I'm not sure that `Template.collTest` is guaranteed to exist...